UsingLinkClicky.md

```diff
@@ -0,0 +1,396 @@
+
+### More on Tags
+
+I love using tags. I can tag one test with my initials, then using the --tags <my initials>, I can run just that one test. Tags provide lots of flexibility. However, our features and scenarios can drown in tag soup if we aren’t careful. I’m proposing we adhere to a strict approach for using tags. Here’s what I’m proposing:
+
+#### Lifecycle Tags - *Required for every scenario*
+
+All scenarios should be tagged with a single lifecycle tag. 
+
+<table>
+  <tr>
+    <td>@dev </td>
+    <td>Only for development tests. Not safe for production.</td>
+  </tr>
+  <tr>
+    <td>@deploy</td>
+    <td> Applies tests specific for a freshly deployed website. Not useful once production begins.</td>
+  </tr>
+  <tr>
+    <td>@safe</td>
+    <td>Useful for tests that can work on dev, deploy, production, launch, and live sites. These tests may not create any content, change any settings, or leave any trace that it has run. Note: some Scenarios tagged as @safe clear the cache. We have labeled clearing the cache as a "safe" operation.</td>
+  </tr>
+  <tr>
+    <td>@launch</td>
+    <td>Useful for testing that placeholder content has been removed.</td>
+  </tr>
+  <tr>
+    <td>@live</td>
+    <td>Applies only for sites post launch. These tests verify launch tasks such as verifying meta-tags, vhost URLs, removing "Request to Launch" menu item, etc.</td>
+  </tr>
+</table>
+
+
+#### Additional Tags
+
+Along with lifecycle, tags can identify other situations.
+
+<table>
+  <tr>
+    <td>@destructive</td>
+    <td>Only for development tests. This scenario may change configuration or leave lasting items/content/config on the site it is running on. Not safe for production. </td>
+  </tr>
+  <tr>
+    <td></td>
+    <td>Authentication - Under consideration</td>
+  </tr>
+  <tr>
+    <td>@saml</td>
+    <td></td>
+  </tr>
+  <tr>
+    <td>@webauth</td>
+    <td></td>
+  </tr>
+  <tr>
+    <td></td>
+    <td>Hosting - Under consideration</td>
+  </tr>
+  <tr>
+    <td>@sites</td>
+    <td></td>
+  </tr>
+  <tr>
+    <td>@local</td>
+    <td></td>
+  </tr>
+  <tr>
+    <td>@anchorage</td>
+    <td></td>
+  </tr>
+</table>
+
+
+#### Considerations for Tags
+
+* Unless a scenario is tagged with @destructive it is assumed it will not make destructive changes to the site. 
+
+* All scenarios should have exactly one lifecycle tag: (@dev, @deploy, @safe, @launch, and @live)
+
+[Learn about d.o tags](https://www.drupal.org/node/1780048)
+
+### More on Suites
+
+Suites target the various phases of the product and site life cycles. Here are the suite definitions I’m proposing:
+
+<table>
+  <tr>
+    <td>Suite</td>
+    <td>Runs only...</td>
+    <td>Excludes (~)</td>
+  </tr>
+  <tr>
+    <td>default</td>
+    <td>@safe  </td>
+    <td>@destructive</td>
+  </tr>
+  <tr>
+    <td>dev</td>
+    <td>Everything but >></td>
+    <td>@live @launch</td>
+  </tr>
+  <tr>
+    <td>deploy</td>
+    <td>@safe @deploy </td>
+    <td>@destructive @dev @launch @live</td>
```

Comment:
Do we need to explicitly exclude live? If so we should explicitly exclude launch as well.